CDCgov/NEDSS-Modernization
# CNFT1-1463 Comparable Investigations

## Description

Adds a comparable property to Investigations so that only Investigations created through Page Builder may be Compared.

## Tickets

* [CNFT1-1463](https://cdc-nbs.atlassian.net/browse/CNFT1-1463)

## Steps to verify

1. Add a `2019 Novel Coronavirus` Investigation
2. Return to the Patient Profile
3. Add another `2019 Novel Coronavirus` Investigation
4. Return to the Patient Profile
5. Select the two `2019 Novel Coronavirus` Investigations
6. Click "Compare Investigations"
7. Expected to navigate to the Compare Investigations page

![image](https://github.com/CDCgov/NEDSS-Modernization/assets/124325935/4c98c3d6-c384-418d-b964-490e8575575e)

---

1. Search for a Patient
2. Navigating to the Patient Profile
3. Add a `Diptheria` Investigation (Legacy)
4. Return to the Patient Profile
5. Add another `Diptheria` Investigation
6. Return to the Patient Profile
7. Notice that the two `Diptheria` Investigations cannot be compared

![image](https://github.com/CDCgov/NEDSS-Modernization/assets/124325935/dfee29eb-c7b8-4a6d-a0f7-f42984302553)




[CNFT1-1463]: https://cdc-nbs.atlassian.net/browse/CNFT1-1463?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
